mail/mail.go

```diff
@@ -15,30 +15,31 @@ type Client struct {
 	Config config.Config
 }
 
-func (mc *Client) SendEmail(ctx context.Context, email, message string) error {
+func (mc *Client) SendEmail(ctx context.Context, email, subject, message string) error {
 	debug, ok := ctx.Value(settings.DebugKey).(bool)
 	if ok && debug {
-		return mc.SendEmailStub(email, message)
+		return mc.SendEmailStub(email, subject, message)
 	}
-	return mc.SendSMTPEmail(email, message)
+	return mc.SendSMTPEmail(email, subject, message)
 }
 
-func (mc *Client) SendEmailStub(email, message string) error {
+func (mc *Client) SendEmailStub(email, subject, message string) error {
```

Comment:
just don't quite get it, why do you need this stub if you already have real working implementation? To only call it when debug mode is enabled?
And why do you test it then separately? https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/pull/4/files#diff-68ab1d1f6346e32650981f75b448b629466f5214cf84ac497b0f12ba0197cafaR12